### PR TITLE
#162606983 As a PM, I want to receive an email every time a client is sent the "ultimatedone" script by the bot.

### PR DESCRIPTION
**Pull request by Tony-Ndichu**
#### What does this PR do?

This PR adds functionality to send an email to the Product Manager once a client is sent the "ultimatedone" script by the bot.

#### Description of Task to be completed

Once a user's topic is set to "ultimatedone", an email containing the user's id and messages sent/received by him/her should be sent to the PM's email

#### How should this be manually tested?

- $ git clone https://github.com/IDEOorg/steps-bot.git
- $ cd steps-bot
- $ git checkout ft-send-PM-email-when-ultimatedone-162606983
- $ npm install
- $ npm start
- from the (twiliogui)[http://twilio-gui.herokuapp.com] app reply to the bot with a `B` to complete the task

#### Any background context you want to provide?

Initially, no email was sent once a user's status was set to 'ultimatedone'

#### What are the relevant pivotal tracker stories?

[#162606983](https://www.pivotaltracker.com/story/show/162606983)

#### Screenshots (if appropriate)

screenshot 2019-01-25 at 11 06 05

### CODE REVIEW

File: src/PmEmail.js [hunk 1]
```diff
@@ -0,0 +1,391 @@
+module.exports = {
```
Comment (PriscillaSam) [inline src/PmEmail.js]: Hi @Tony-Ndichu. It will be good to have an email templates folder that will hold all email templates.

**Comment by PriscillaSam** [comment]
This looks well-done @Tony-Ndichu. 👍

File: src/PmEmail.js (hunk 1 above)
Comment (Tony-Ndichu) [reply src/PmEmail.js]: I agree @PriscillaSam , I've hence refactored the codebase to include the changes.

File: src/Updater.js [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
-const sgMail = require("@sendgrid/mail");
+require('dotenv').config();
+const api = require('./api');
+const moment = require('moment');
+const sgMail = require('@sendgrid/mail');
+const pmEmail = require('./email_templates/PmEmail');
 
 sgMail.setApiKey(process.env.SENDGRID_API_KEY);
 
+let messages;
```
[the remark this answers is not in the document]
Comment (Tony-Ndichu) [reply src/Updater.js]: @micahoriaso , thanks for the tip, I have removed the variable.

File: src/Updater.js [hunk 3]
```diff
@@ -320,3 +326,28 @@ function sendHelpEmailToCoach(
       });
     });
 }
+
+function sendUltimateDoneEmailToPm(client) {
+  const title = 'Your client <a href="' + process.env.ADMIN_URL + '/clients/' + client.id + '/tasks' + `"><strong>${client.first_name} ${client.last_name}&nbsp;</strong></a> has completed all of their action items!`;
```
[the remark this answers is not in the document]
Comment (Tony-Ndichu) [reply src/Updater.js]: @andela-kakpobome thankyou for the feedback, I have made the necessary changes

**Pull request merged**